src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/ElasticSearchAnalysisGraph.java

```diff
@@ -332,33 +329,22 @@ public void construct() {
     pluginController.initPlugins();
   }
 
-  private void admissionControlRca() {
-    Metric acCurrent = new AdmissionControl_CurrentValue(EVALUATION_INTERVAL_SECONDS);
-    Metric acThreshold = new AdmissionControl_ThresholdValue(EVALUATION_INTERVAL_SECONDS);
-    Metric acRejectionCount = new AdmissionControl_RejectionCount(EVALUATION_INTERVAL_SECONDS);
-
-    acCurrent.addTag(TAG_LOCUS, LOCUS_DATA_MASTER_NODE);
-    acThreshold.addTag(TAG_LOCUS, LOCUS_DATA_MASTER_NODE);
-    acRejectionCount.addTag(TAG_LOCUS, LOCUS_DATA_MASTER_NODE);
-
-    addLeaf(acCurrent);
-    addLeaf(acThreshold);
-    addLeaf(acRejectionCount);
-
-    AdmissionControllerRca admissionControllerRca = new AdmissionControllerRca(
-            RCA_PERIOD, acCurrent, acThreshold, acRejectionCount);
-    admissionControllerRca.addTag(TAG_LOCUS, LOCUS_DATA_MASTER_NODE);
-    admissionControllerRca.addAllUpstreams(Arrays.asList(acCurrent, acThreshold, acRejectionCount));
+  private AdmissionControlDecider buildAdmissionControlDecider(Metric heapUsed, Metric heapMax) {
+    AdmissionControlRca admissionControlRca = new AdmissionControlRca(RCA_PERIOD, heapUsed, heapMax);
+    admissionControlRca.addTag(TAG_LOCUS, LOCUS_DATA_NODE);
+    admissionControlRca.addAllUpstreams(Arrays.asList(heapUsed, heapMax));
```

Comment:
Trying to understand why we changed the input metrics to `AdmissionControlRca`. 
Previously we were using Admission Control current value, threshold and rejection count. Now, we are using only heap metrics. 

What happens if the heap usage is spiky ? Even with `coolOffPeriod`, we can end up suggesting rejection threshold change frequently and drastically, where we jump from a lower heap-range bucket to a higher bucket. This will impact the heap stability and the wider customer experience.

Reply:
Previous metrics were unused. Will add new metrics as required.
Suggestion on avoiding large threshold diff on drastic heap range change?

Reply:
For frequency, we can extend the cool-off time period to a larger value and for flip-flop we check for delta change suggested ? 

Again, I still have my doubts on the efficiency of the RCA to suggest Admission Control Threshold change just based off raw JVM metrics.

Reply:
We can change default cool-off period to 15 mins instead of 5 and later tune it as required based on case studies.

Reply:
Changed `DEFAULT_COOL_OFF_PERIOD_IN_MILLIS` to 15 mins in `AdmissionControlAction`